licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/postDecision/CaTaskListSpec.groovy

```diff
@@ -165,4 +183,37 @@ class CaTaskListSpec extends GebReportingSpec {
     then: 'I see the BASS offer page'
     at BassOfferPage
   }
+
+  def 'Shows correct button labels for tasks when DM previously refused HDC'() {
+    given: 'A refused licence'
+    testData.loadLicence('decision/refused')
+
+    when: 'I view the task list'
+    to TaskListPage, testData.markAndrewsBookingId
+
+    then: 'I see the task buttons and the submit button'
+    taskListActions.size() == 9
+
+    and: 'All editable tasks have View/Edit buttons'
+    taskListActions.take(7).every { it.text() == 'View/Edit' }
+
+    and: 'Final checks task is view only'
+    taskListAction(tasks.finalChecks).text() == 'Change'
+
+    and: 'Resubmit to DM is available'
+    taskListAction(tasks.resubmitToDm).text() == 'Resubmit'
+  }
+
+  def 'Shows the correct button labels for tasks when DM previously refused HDC'() {
```

Comment:
this test has the same name as the test above?

Reply:
oops, corrected